Gemfile

```diff
@@ -168,6 +168,7 @@ group :test do
   # Rails performance tests
   gem 'rails-perftest'
   gem 'rspec-collection_matchers' # Provides matchers for dealing with arrays
```

Comment:
Added because the test was running alarmingly slowly, and I wanted to find the bottleneck.